app/models/medical_history.rb

```diff
@@ -1,3 +1,6 @@
 class MedicalHistory < ApplicationRecord
   blongs_to :policy
+
+  #validation
+  validates :height, :weight, :abd, :chest, :identification_mark, :date, :operation_detail, presence: true
```

Comment:
@VaibhaviJoshi-BTC : Policy holder may not know all these details at the time of policy , 
cross check all the validations in all models